package.json

```diff
@@ -37,33 +42,35 @@
     ]
   },
   "devDependencies": {
-    "babel-cli": "6.x",
-    "babel-core": "6.x",
-    "babel-preset-shopify": "15.0.1",
-    "eslint": "^3.19.0",
-    "eslint-index": "^1.3.0",
+    "babel-cli": "^6.24.1",
+    "babel-core": "^6.25.0",
+    "babel-preset-shopify": "^16.1.0",
+    "eslint": "^4.3.0",
+    "eslint-index": "^1.4.0",
     "eslint-plugin-shopify": "file:.",
-    "isparta": "4.x",
-    "istanbul": "0.4.4",
-    "mocha": "3.x"
+    "isparta": "^4.0.0",
+    "istanbul": "^0.4.5",
+    "mocha": "^3.4.2",
+    "react": "^15.6.1",
+    "react-dom": "^15.6.1"
   },
   "peerDependencies": {
-    "eslint": "<4 >=3.17"
+    "eslint": "<5 >=4.3.0"
   },
   "dependencies": {
     "babel-eslint": "^7.2.3",
-    "eslint-plugin-ava": "^4.2.0",
-    "eslint-plugin-babel": "^4.1.1",
-    "eslint-plugin-chai-expect": "1.1.x",
-    "eslint-plugin-flowtype": "^2.32.1",
-    "eslint-plugin-import": "2.2.x",
-    "eslint-plugin-jsx-a11y": "^5.0.1",
-    "eslint-plugin-lodash": "^2.4.2",
-    "eslint-plugin-mocha": "^4.9.0",
-    "eslint-plugin-node": "^4.2.2",
+    "eslint-plugin-ava": "^4.2.1",
```

Comment:
Should we allow patch updates on all our plugins? Previously some weren't and some were. Do we remember the reasoning behind that?